win32/Makefile.sub

```diff
@@ -1418,6 +1418,9 @@ gc_impl.obj: $(BUILTIN_GC_PATH)
 lex.c: {$(srcdir)}lex.c.blt
 	copy $(?:/=\) $@
 
+rjit_funcs.c: {$(srcdir)}rjit_funcs.c.template
+	copy $(?:/=\) $@
```

Comment:
RJIT (and other JIT implementations using this feature) is going to unsupport Windows.

Reply:
I'm not sure yet. It only works on platforms that support YJIT right now, and I'm not sure if YJIT supports windows. I will investigate

Reply:
YJIT doesn't support windows.